Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -628,6 +637,48 @@ void GameList::OpenGCSaveFolder()
     ModalMessageBox::information(this, tr("Information"), tr("No save data found."));
 }
 
+#ifdef _WIN32
+void GameList::AddShortcutToDesktop()
+{
+  const auto game = GetSelectedGame();
+  const auto game_name = game->GetLongName();
```

Comment:
```suggestion
  const auto& game_name = game->GetLongName();
```
Otherwise, you're copying the name for no reason. The next two lines need to be fixed too.